src/gui/activitywidget.h

```diff
@@ -87,10 +91,17 @@ private slots:
     QPushButton *_copyBtn;
 
     QSet<QString> _accountsWithoutActivities;
-    QMap<int, NotificationWidget*> _widgetForNotifId;
+    QMap<Activity::Identifier, NotificationWidget*> _widgetForNotifId;
     QElapsedTimer _guiLogTimer;
     QSet<int> _guiLoggedNotifications;
-    int _notificationRequests;
+    ActivityList _blacklistedActivities;
```

Comment:
Yes, we present the notifications from the server again, IF the notification is of the special form that has no action, ie. no button defined. I think that is the desired behaviour, as the admin will disable the notifification if it is not longer important.